warehouse/query-core/src/main/java/datawave/next/stats/StatUtil.java

```diff
@@ -13,10 +15,34 @@ public class StatUtil {
      * @return the formatted time
      */
     public static String formatNanos(long ns) {
-        long millis = ns / 1_000_000;
-        if (millis > 0) {
+        StringBuilder sb = new StringBuilder();
+        return sb.append(format(TimeUnit.NANOSECONDS, TimeUnit.HOURS, ns, 0)).append(format(TimeUnit.NANOSECONDS, TimeUnit.MINUTES, ns, 60))
+                        .append(format(TimeUnit.NANOSECONDS, TimeUnit.SECONDS, ns, 60)).append(format(TimeUnit.NANOSECONDS, TimeUnit.MILLISECONDS, ns, 1000))
+                        .toString();
```

Comment:
```suggestion
        // @formatter:off
        return new StringBuilder()
                        .append(format(TimeUnit.NANOSECONDS, TimeUnit.HOURS, ns, 0))
                        .append(format(TimeUnit.NANOSECONDS, TimeUnit.MINUTES, ns, 60))
                        .append(format(TimeUnit.NANOSECONDS, TimeUnit.SECONDS, ns, 60))
                        .append(format(TimeUnit.NANOSECONDS, TimeUnit.MILLISECONDS, ns, 1000))
                        .toString();
        // @formatter:on
```

Inlines the StringBuilder and formats the calls to append to make them more readable.